app/helpers/application_helper/button/test_button.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationHelper::Button::TestButton < ApplicationHelper::Button::Basic
+  needs_record
+  def initialize(view_context, view_binding, instance_data, props)
+    super(view_context, view_binding, instance_data, props)
+    @method = props[:options][:method]
+  end
+
+  def skip?
+    return true unless @method
+    return !@record.send("supports_#{@method}?")
```

Comment:
Shrug.. feels like supports_feature_mixin is not that useful if we have to test for method existence before calling..